### PR TITLE
Add Github org, merge platformsOther into Unofficial section, readjust columns

### PR DESCRIPTION
**Pull request by crertel**
## Introduction

Inspired by the #1438 discussion, I'd like to submit this as a possible change set.

I'd like feedback from the marketing folks (@tomberek , @djacu , and @garbas , I think?)--and if this doesn't spark joy or isn't something marketing likes, please by all means close the PR! Not trying to make more work for you all.

(Also, this is my first time working with Astro--I _think_ I did this correctly, but please let me know what I can do differently if it's wrong.)

## Proposed changes

* Add the Github organization to the Official Spaces section
* Move the `otherPlatforms` links down into Unofficial Spaces
* Update the columns for Official Spaces to take up space from the otherPlatforms move
* Remove the `otherPlatforms` affordance

## Screenshot
image

### CODE REVIEW

**Comment by thilobillerbeck** [comment]
Hi, thanks for putting work into this. <3 I'll try to do an in depth review soon. Just a few glances:

- the issue discussion came up with a separate social media and platform section. It would be great if you could realize this. If you don't feel comfortable enough with the stack yet, no problem, we can do that in a second step. :)
- have you tried to lay out the official platforms as a row rather than columns? Would go more in line with the other parts.
- We should tone down the unofficial platforms. Some suggestions would be blue color on white text or maller font size

**Comment by crertel** [comment]
@thilobillerbeck how about this?

image

Thoughts:

* I split `CommunityLink` into `OfficialCommunityLink` and `UnofficialCommunityLink`
* I used `bg-nix-blue-light`, and used `bg-nix-blue` for the hover state (basically, just one step lighter. The alternative would be to add a `nix-blue` border and text.

What do you think? Want me to go further and make it blue-on-white?

**Comment by crertel** [comment]
Also, re: the social media section, which things go in there vs the unofficial space?

I feel comfortable making the change, but just need to know what goes where--that would give us three sections, right?

**Comment by crertel** [comment]
Some variants.

## White background, blue text

image


## White background, blue text, border-2

image

**Comment by crertel** [comment]
Just for shits-n-giggles, here's what a 5-column layout looks like:

image

**Comment by crertel** [comment]
Also, tweaks to make mobile 1-column:

image


image

**Comment by crertel** [comment]
(holding off on pushing up commits until y'all pick a variation you like :) )

**Comment by AshleyYakeley** [comment]
Wasn't there a plan for a three-way split?

> * official moderation (Matrix, Discourse, GitHub, ?Zulip?)
> 
> * official communication (Mastodon, Twitter)
> 
> * unofficial resources (StackOverflow, Discord, Reddit, IRC, etc)

**Comment by crertel** [comment]
@thilobillerbeck @tomberek which of those variations do you want me to continue with? I think the blue-on-white with extralightblue onhover is best. I think we can leave out the 5-column one.

Also, how many sections do we need, what are they named, and what goes in them?

**Comment by tomberek** [comment]
> I think the blue-on-white with extralightblue onhover is best. I think we can leave out the 5-column one

@crertel : that looks good. There is the differentiation in both size and color. It's better than what we have now. Regarding the 3-way split, let's not block on that - that can be another PR (i don't have a strong opinion on it being same PR or not).

Thank you for also checking mobile. Let us know when it is in a state you like and ready to merge.

**Comment by nat-418** [comment]
Two questions:

1. Can this PR integrate the unofficial channels from the Wiki (e.g. Telegram)?

2. Can this PR provide some way for people to add unofficial channels?

I think the goal of "adding more resources" raised in the relevant issue is going to hit a ceiling in terms of contribution friction that the Wiki does not suffer, and it probably makes more sense to send people there via a link, similarly to how we link to commercial support in this page.

**Comment by crertel** [comment]
@tomberek perfect, I'll get those changes up right quick--just finished working a surprise memorial day incident. :(

File: src/pages/community.astro
Comment (crertel) [inline src/pages/community.astro:342]: IIRC this was just fixing some mismatched indentation.

File: src/pages/community.astro
Comment (crertel) [inline src/pages/community.astro:174]: Set to single column on mobile, prevents text jamming.

File: src/pages/community.astro
Comment (crertel) [inline src/pages/community.astro:177]: This removes the `platformsOther`, since those options are (for now) merged into `platformsOutside`.

This also makes the DOM mirror what we do for the unofficial links, instead of the older solution which was done to make a 2-3 layout on mobile--we just do a single column now.

File: src/pages/community.astro
Comment (crertel) [inline src/pages/community.astro:90]: `platformsOther` -> `platformsOutside` merge.

File: src/pages/community.astro
Comment (crertel) [inline src/pages/community.astro:101]: `platformsOther` -> `platformsOutside` merge.

File: src/pages/community.astro
Comment (crertel) [inline src/pages/community.astro:14]: Community link subtype split.

File: src/pages/community.astro
Comment (crertel) [inline src/pages/community.astro:87]: Seems weird we didn't have the Github org lol.

File: src/components/pages/community/UnofficialCommunityLink.astro
Comment (crertel) [inline src/components/pages/community/UnofficialCommunityLink.astro:1]: Community link subtype split.

**Review by crertel — COMMENTED** [review]
I believe this is ready to go now @tomberek !

**Comment by crertel** [comment]
Friendly bump @tomberek @thilobillerbeck . Anything else I can do on this?

**Review by tomberek — APPROVED** [review]
Thanks. Sorry for the delayed response.

**Comment by crertel** [comment]
> Thanks. Sorry for the delayed response.

Not a problem at all--I'm a maintainer on some other things, and I know the drill. :)

**Pull request merged**
